src/Http/HttpServiceProvider.php

```diff
@@ -62,4 +62,25 @@ public function register()
             return new SlugManager($this->app->make('flarum.http.selectedSlugDrivers'));
         });
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function boot()
+    {
+        $this->setAccessTokenTypes();
+    }
+
+    protected function setAccessTokenTypes()
```

Comment:
Let's make this a container binding in `register`; the difference for this PR wouldn't be that significant, and we'd need this anyway later for the extender.

Reply:
The type array I mean; the static assignment should be where it is now. I'd even argue to put it directly in boot.

Reply:
This is a copy-paste of how post types work. I don't mind changing it, but then it would be logical to also change it for post types.